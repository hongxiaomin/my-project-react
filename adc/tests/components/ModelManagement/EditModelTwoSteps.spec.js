import React from 'react'
import { shallow } from 'enzyme'
import { TwoStep } from 'routes/ModelManagement/routes/ModelManagement/components/EditModel/TwoSteps'

describe('(Component) TwoStep', () => {
  const that = { state: { interation: 11111, modelName: '1', description: '1' } }
  const process = { id: 1, name: '1' }
  const product = { id: 1, name: '1' }
  const defect = { id: 1, name: '1' }
  const data = {}
  const imageSource = {}
  const dispatch = jest.fn()
  const sourceType = 'filter'
  const wrapper = shallow(
    <TwoStep
      that={that}
      process={process}
      product={product}
      defect={defect}
      data={data}
      imageSource={imageSource}
      dispatch={dispatch}
      sourceType={sourceType}
    />,
  )
  it('render normally', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
