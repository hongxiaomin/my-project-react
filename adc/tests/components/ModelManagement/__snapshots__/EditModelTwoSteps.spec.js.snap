// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) TwoStep render normally 1`] = `
<div
  className="twoStep"
>
  <div
    className="dialogBody"
  >
    <Tabs
      activeKey="3"
      hideAdd={false}
      onChange={[Function]}
      prefixCls="ant-tabs"
    >
      <TabPane
        placeholder={null}
        tab="Image Path"
      >
        <div>
          <span
            style={
              Object {
                "paddingLeft": "15px",
              }
            }
          >
            Input Image Pathï¼š
          </span>
          <Input
            defaultValue=""
            disabled={false}
            onChange={[Function]}
            prefixCls="ant-input"
            style={
              Object {
                "margin": "30px 0px",
                "width": "200px",
              }
            }
            type="text"
          />
        </div>
      </TabPane>
      <TabPane
        placeholder={null}
        tab="Filter"
      >
        <Filter
          activeKey="3"
          data={Object {}}
          defect={
            Object {
              "id": 1,
              "name": "1",
            }
          }
          dispatch={[Function]}
          process={
            Object {
              "id": 1,
              "name": "1",
            }
          }
          product={
            Object {
              "id": 1,
              "name": "1",
            }
          }
          that={
            Object {
              "state": Object {
                "description": "1",
                "interation": 11111,
                "modelName": "1",
              },
            }
          }
        />
      </TabPane>
    </Tabs>
  </div>
</div>
`;
